portfolio/src/main/webapp/script.js

```diff
@@ -76,7 +76,9 @@ function hasSlides() {
 
 /** Fetches commenta from the server and adds them to the DOM. */
 function getComments() {
-  var url = '/data?quantity=10&filter=' + document.getElementById("filter").value;
+  /*var url = '/data?quantity=' + document.getElementById("quantity") +
+            '&filter=' + document.getElementById("filter").value;*/
+  var url = '/data?quantity=3&filter=' + document.getElementById("filter").value;
```

Comment:
Please use `let` over `var` (see style guide and javascript.info). Both will work here, but `let` behaves in a much more predictable way (as e.g. with regards to the scope where the variable is visible).